sqldb/sqlc/migrations/000004_invoice_expiry_fix.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Update the expiry to 3600 seconds (1 hour) for all records in the invoices
+-- table. This is needed as previously we stored raw time.Duration values which
+-- are 64 bit integers and are used to express duration in nanoseconds however
+-- the intent is to store invoice expiry in seconds.
+UPDATE invoices SET expiry = 3600;
```

Comment:
> one thing to note is that 3600 (1 hour ) is not the default used by LND. LND uses 24 hours (86400 seconds). See DefaultInvoiceExpiry (the one defined in invoicesrpc, not the one in zpay32)

Done, missed this, thank you!

> So cant it happen that perhaps any valid, unexpired, invoices currently in memory in the expiry watcher actually then dont get reloaded on startup since their expiry has been shortened significantly?

It can happen that we shorten intended expiries, but I'm not sure that we can do anything better. Thankfully the number of affected users and invoices should be pretty low since we don't have full migration of invoices yet, so only some new nodes are affected (who choose to use the native schema).